RELEASE.md

```diff
@@ -42,3 +42,28 @@ If bug fixes need to be backported to previous releases, a PR through main is al
     - `release-vx.y.1 <- main`
 3. Create a PR to cherry-pick the fix into `release-vx.y.2`.
     - `release-vx.y.2 <- main`
+
+## Security
+
+Tip of the branch from which runtime release is considered should be signed at least by 2 signatures.
```

Comment:
Is this some convention in other parachains or arbitrary ?

Reply:
Other chains do not use nix, and cannot do deterministic builds. They are unsecure by design. So they likely talk to each other during release in Matrix chats. 

This proposal allows to make security which is on chain == Security off chain. At least nominally. Other just cannot do it without nix.

Reply:
Look into mention of `crev` in our chats. Recent mentions will be with nix and git tags and dotsama indentity. 

Imho this PR is last missing step in possible security in dotsama.

Reply:
At least nominally :)

Reply:
Without something verifying the signatures automatically for deployment, the security is just an assumption IMO

Reply:
also i proved this works in basic bash example.

Reply:
Just curious, What problem/requirement are we trying to solve anyway?

Reply:
Security as low as its lowest element in chain. We have ability to full multisignature security on chain and in vendoing code. 

But in git we have 0 signature security with ability of Mergifyio or GitHub teams to put anycode they want into our release. So our CI and hosting is not secure at all. 

SRE may like to push secure CI agenda, but that just shift lower security point from them to SRE. The only option to fix that link to multisig.

Reply:
In this case git/CI security is same as multisig on testnet at least.

Reply:
I think signatures is cheaper and safer alternative than building badly working non secured (because centralized) CI. @bostercf Bret do you agree?

Reply:
closing until further comments